include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -621,9 +624,148 @@ struct SPhysicalDeviceFeatures
     
     /* Extensions Exposed as Features: */
 
+    /* VK_AMD_buffer_marker */
+    /* VK_AMD_calibrated_timestamps */
+    /* VK_AMD_display_native_hdr */
+    /* VK_AMD_draw_indirect_count */
+    /* VK_AMD_gcn_shader */
+    /* VK_AMD_gpa_interface */
+    /* VK_AMD_gpu_shader_half_float */
+    /* VK_AMD_gpu_shader_half_float_fetch */
+    /* VK_AMD_gpu_shader_int16 */
+    /* VK_AMD_image_layout_resolve */
+    /* VK_AMD_memory_overallocation_behavior */
+    /* VK_AMD_mixed_attachment_samples */
+    /* VK_AMD_negative_viewport_height */
+    /* VK_AMD_pipeline_compiler_control */
+    /* VK_AMD_programmable_sample_locations */
+    /* VK_AMD_rasterization_order */
+    /* VK_AMD_shader_ballot */
+    /* VK_AMD_shader_explicit_vertex_parameter */
+    /* VK_AMD_shader_fragment_mask */
+    /* VK_AMD_shader_image_load_store_lod */
+    /* VK_AMD_shader_info */
+    /* VK_AMD_shader_trinary_minmax */
+    /* VK_AMD_texture_gather_bias_lod */
+    /* VK_AMD_wave_limits */
+    /* VK_ANDROID_external_memory_android_hardware_buffer */
+    /* VK_ANDROID_native_buffer */
+    /* VK_EXT_2d_imageview_3d_image */
+    /* VK_EXT_calibrated_timestamps */
+    /* VK_EXT_debug_marker */
+    /* VK_EXT_debug_report */
+    /* VK_EXT_debug_utils */
+    /* VK_EXT_depth_clip_control */
+    /* VK_EXT_depth_clip_enable */
+    /* VK_EXT_depth_range_unrestricted */
+    /* VK_EXT_device_memory_report */
+    /* VK_EXT_display_control */
+    /* VK_EXT_external_memory_dma_buf */
+    /* VK_EXT_filter_cubic */
+    /* VK_EXT_full_screen_exclusive */
+    /* VK_EXT_global_priority_query */
+    /* VK_EXT_hdr_metadata */
+    /* VK_EXT_image_drm_format_modifier */
+    /* VK_EXT_load_store_op_none */
+    /* VK_EXT_memory_priority */
+    /* VK_EXT_metal_objects */
+    /* VK_EXT_non_seamless_cube_map */
+    /* VK_EXT_physical_device_drm */
+    /* VK_EXT_pipeline_creation_feedback */
+    /* VK_EXT_post_depth_coverage */
+    /* VK_EXT_queue_family_foreign */
+    /* VK_EXT_rgba10x6_formats */
+    /* VK_EXT_separate_stencil_usage */
+    /* VK_EXT_shader_module_identifier */
+    /* VK_EXT_shader_stencil_export */
+    /* VK_EXT_swapchain_colorspace */
+    /* VK_EXT_tooling_info */
+    /* VK_EXT_validation_cache */
+    /* VK_EXT_vertex_input_dynamic_state */
+    /* VK_EXT_video_decode_h264 */
+    /* VK_EXT_video_decode_h265 */
+    /* VK_EXT_video_encode_h264 */
+    /* VK_EXTX_portability_subset */
+    /* VK_GOOGLE_decorate_string */
+    /* VK_GOOGLE_display_timing */
```

Comment:
amazing extension, gonna put it on the immediate TODO shortlist, enables construction much better presentation oracles and much better frame pacing